### PR TITLE
fix alloy-secret naming

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Fix alloy-secret naming

* Renaming secret to alloy-logging-secret
* Prefix secret with the cluster name
* Fix kyverno policy exception namespace

### CODE REVIEW

File: pkg/resource/alloy-secret/alloy-logging-secret.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
```
Comment (QuentinBisson) [inline pkg/resource/alloy-secret/alloy-logging-secret.go:14]: Is thé app not expecting clusterId-logging-secret? Why thé need to add alloy?

File: pkg/resource/logging-config/alloy/logging-config.yaml.template [hunk 2]
```diff
@@ -46,3 +46,7 @@ alloy:
       - name: runlogjournal
         hostPath:
           path: /run/log/journal
+kyvernoPolicyExceptions:
+  enabled: true
+  # Ensure the kyverno policy exceptions are created in the namespace where alloy is installed
+  namespace: null
```
Comment (QuentinBisson) [inline pkg/resource/logging-config/alloy/logging-config.yaml.template:52]: Why? I think they need to bé in the giantswarm Namespace for now
```suggestion
  namespace: null
```

File: pkg/resource/alloy-secret/alloy-logging-secret.go (hunk 1 above)
Comment (TheoBrigitte) [reply pkg/resource/alloy-secret/alloy-logging-secret.go:14]: Yes the final format is `$clusterID-alloy-logging-secret`. I added the `logging` part to make it explicit that this secret is intended for logging usage.

File: pkg/resource/logging-config/alloy/logging-config.yaml.template (hunk 2 above)
Comment (TheoBrigitte) [reply pkg/resource/logging-config/alloy/logging-config.yaml.template]: You're right this is not needed, removing it.

**Review by QuentinBisson — APPROVED** [review]

**Pull request merged**